src/main/java/com/example/notes/controller/NoteController.java

```diff
@@ -22,4 +20,10 @@ NoteDtoList getAllNotes() {
         return noteService.getAllNotes();
     }
 
+    @PostMapping("/create")
+    @ResponseStatus(HttpStatus.CREATED)
+    NoteDto createNote(@RequestBody NoteDto noteDto) {
```

Comment:
Replace the NoteDto with the CreateNoteInput dto.
This dto should only contain the content field. The createdAt can be saved inside of the entity on the service layer before the entity is saved.

Reply:
Done